domcfghelper.py

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/python
```

Comment:
`#!/usr/bin/python2` 
because your script will not work on systems where default Python is Python3